### PR TITLE
feat(eventloop utilization): adds event loop utilization metric

### LINKED ISSUE #65: use eventLoopUtilization if it is available
## 🚀 Feature Proposal

Node v14.10.0 shipped a more precise way to check on the event loop utilization:
https://nodejs.org/api/perf_hooks.html#perf_hooks_performance_eventlooputilization_utilization1_utilization2

This module should use that if it is available.

### PR DESCRIPTION
**Pull request by sujeshthekkepatt**
Nodejs introduced eventLoopUtilization api for measuring eventloop utilization.
This PR addresses the concerns raised in #65.

Closes #65

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -52,6 +53,49 @@ test('Should return 503 on maxEventLoopDelay', t => {
   })
 })
 
+test('Should return 503 on maxEventloopUtilization', t => {
+  const isSupportedVersion = satisfies(valid(coerce(process.version)), '12.19.0 || >=14.0.0')
```
Comment (mcollina) [inline test.js:57]: you can pass a `skip` option to tap, use that instead

**Review by mcollina — APPROVED** [review]
lgtm

File: test.js (hunk 1 above)
Comment (sujeshthekkepatt) [reply test.js:57]: Sure will change that :)

**Review by zekth — APPROVED** [review]
lgtm

File: test.js (hunk 1 above)
Comment (sujeshthekkepatt) [reply test.js]: I have added te `skip` option instead of `end` @mcollina

**Pull request merged**